src/client/datascience/commands/commandRegistry.ts

```diff
@@ -322,23 +324,23 @@ export class CommandRegistry implements IDisposable {
     private async debugStepOver(): Promise<void> {
         // Make sure that we are in debug mode
         if (this.debugService.activeDebugSession) {
-            this.commandManager.executeCommand('workbench.action.debug.stepOver');
+            void this.commandManager.executeCommand('workbench.action.debug.stepOver');
```

Comment:
I think we should await on these, as this method is async & the calling method is also async.

Reply:
Maybe? It's a functionality change though. Looks like this was just an eslint update. Previously our code would call debugStepOver then fire and forget on the workbench action. void keeps that existing functionality with a thenable. Changing to await changes how debugStepOver returns, might be ok? But might not be what we want.

Reply:
agreed, it could change existing code, 
Though i feel its the right thing to do, as `debugStepOver` is already waiting & the command also completes when its done.
Feels incorrect not to do it... but not something we should address in this PR...
Lets leave it as is